coap/sbom_libcoap.yml

```diff
@@ -1,9 +1,7 @@
 name: libcoap
-version: 4.3.4
+version: 4.3.5-rc2
```

Comment:
This seems to be marked as release candidate, are we expecting any changes in `v4.3.5` ?

Reply:
Also I dont see the submodule for libcoap updated here.